tools/toolchain/Dockerfile

```diff
@@ -4,6 +4,7 @@ ARG CLANG_BUILD="SKIP"
 ARG CLANG_ARCHIVES
 
 WORKDIR /mnt
+RUN echo install_weak_deps=False >> /etc/dnf/dnf.conf
```

Comment:
Would `--setopt=install_weak_deps=False` option for the subsequent `dnf` invocation work? The less build-time layers the better